M/Microbiome/Versions.toml

```diff
@@ -29,4 +29,4 @@ git-tree-sha1 = "a89979a590578dd9dfb372757522dc881a84db76"
 git-tree-sha1 = "274452475b32b4dd54f9abb6e2297e88d5fe9004"
 
 ["0.6.5"]
-git-tree-sha1 = "4e901f0569d9684cc02130d489098cca084848d4"
+git-tree-sha1 = "8653f05a1a4ee373cf13792833801b6a47e98876"
```

Comment:
You should not update this

Reply:
Even though that sha points to something with a `Project.toml` saying it's compatible with DataFrames v22?

Reply:
Yes, this is immutable. You can release 0.6.6 with the fix if you like. Can still merge this PR though, it shouldn't break anything.

Reply:
OK, got it, thanks